src/main/java/ch/admin/bag/covidcertificate/api/Constants.java

```diff
@@ -53,10 +53,12 @@ public class Constants {
     public static final String KPI_TYPE_MASS_REVOCATION_SUCCESS = "mre-s";
     public static final String KPI_TYPE_MASS_REVOCATION_FAILURE = "mre-f";
     public static final String KPI_TYPE_MASS_REVOCATION_REDUNDANT = "mre-r";
+    public static final String KPI_TYPE_VACCINATION_CONVERSION = "vc";
```

Comment:
Es könnten in Zukunft ja auch Umwandlungen von nicht-Impfzertifikate (sondern z.B. Genesungszertifikate) gehen. Daher vielleicht eher "KPI_TYPE_CERTIFICATE_CONVERSION" ="cc"?